backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -255,31 +256,12 @@ public UserInfo getCurrentUserInfo() {
   }
 
   @AuthorizationConfiguration.RequirePermissionManageUsers
-  public List<UserInfo> getUsersInCurrentOrg() {
+  public List<UserIdentityInfo> getUsersInCurrentOrg() {
     Organization org = _orgService.getCurrentOrganization();
-    Map<String, OrganizationRoleClaims> userClaims = _oktaRepo.getAllUsersForOrganization(org);
-    Set<ApiUser> apiUsers = _apiUserRepo.findAllByLoginEmailIn(userClaims.keySet());
-    // will add facilities to their users in a subsequent PR
-    List<Facility> facilities = _orgService.getFacilities(org);
-    Set<Facility> facilitiesSet = new HashSet<>(facilities);
-    Map<UUID, Facility> facilitiesByUUID =
-        facilities.stream().collect(Collectors.toMap(Facility::getInternalId, Function.identity()));
-    return apiUsers.stream()
-        .map(
-            u -> {
-              OrganizationRoleClaims claims = userClaims.get(u.getLoginEmail());
-              boolean allFacilityAccess = claims.grantsAllFacilityAccess();
-              Set<Facility> accessibleFacilities =
-                  allFacilityAccess
-                      ? facilitiesSet
-                      : claims.getFacilities().stream()
-                          .map(facilitiesByUUID::get)
-                          .collect(Collectors.toSet());
-              OrganizationRoles orgRoles =
-                  new OrganizationRoles(org, accessibleFacilities, claims.getGrantedRoles());
-              return new UserInfo(u, Optional.of(orgRoles), isAdmin(u));
-            })
-        .collect(Collectors.toList());
+    final Set<String> orgUserEmails = _oktaRepo.getAllUsersForOrganization(org);
+    final Set<ApiUser> orgApiUsers = _apiUserRepo.findAllByLoginEmailIn(orgUserEmails);
```

Comment:
Working my way back up to the source, I suggest that we could just return these objects. Probably sorted by name, though?

Reply:
I added the sorting, but the frontend was already performing the sorting.  We'll need some kind of sorting if we ever need to  page the user list anyway.